src/getCoinCombination.test.js

```diff
@@ -7,4 +7,82 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test('0 cent', () => {
```

Comment:
Please, provide more informative description what you check in this test (start with 'should...')

Reply:
And fix it in all test descriptions